include/deal.II/lac/constraint_matrix.h

```diff
@@ -1406,6 +1421,8 @@ ConstraintMatrix::add_entry (const size_type line,
   // exists, since we don't want to enter it twice
   Assert (lines_cache[calculate_line_index(line)] != numbers::invalid_size_type,
           ExcInternalError());
+  Assert (!local_lines.size() || local_lines.is_element(column),
```

Comment:
I think this is useful since we always had that locally active DoFs were constrained against locally relevant DoFs and we knew what the locally relevant DoFs were. But here the locally relevant DoFs may extend to the other side of the domain and one has to make sure that they are actually included in the locally relevant index set?